### ISSUE TITLE
log in

### ISSUE THREAD

**Issue by tskinner34z** [def00111/yahoo-mail-notifier#5]
I am logged in to Yahoo!, but the app says not logged in.

**Comment by def00111** [comment]
You have this problem on Firefox or Chrome?

**Comment by tskinner34z** [comment]
Torch Browser (Chromium)
    On Tuesday, May 11, 2021, 09:41:44 AM EDT, def00111 ***@***.***> wrote:  
 
 


You have this problem on Firefox or Chrome?

**Comment by def00111** [comment]
Right-click on the extension icon on the toolbar and "Check Now" does also not work?

![chrome_ZaN1nYKJCL](https://user-images.githubusercontent.com/10849366/117826713-aa68ac80-b270-11eb-9465-6172a70425ea.png)

**Comment by tskinner34z** [comment]
Right-click doesn't work.
    On Tuesday, May 11, 2021, 9:51:45 AM EDT, def00111 ***@***.***> wrote:  
 
 


Right-click on the extension icon on the toolbar and "Check Now" does also not work?
